client/src/components/Spotlight.js

```diff
@@ -0,0 +1,103 @@
+import React, { Component } from 'react';
+import logoImage from './../images'
+import './css/Spotlight.css';
+
+/**
+ * Spotlight/Impact component.
+ */
+class Spotlight extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+          currentTab: 0,
+          spotlightStyle: "hidden", 
+          impactStyle: "hidden", 
+          spotlightContainer: "hidden", 
+          impactContainer: "hidden"
+        }
+        this.onClickItem = this.onClickItem.bind(this);
+        this.setTab = this.setTab.bind(this);
+        this.setStyles = this.setStyles.bind(this);
+    }
+
+    onClickItem(position) {
+        // TODO: Show detailed information about chosen icon
+        alert("See more information about feature " + position);
+    }
+
+    setTab(tab) {
+      this.setState({currentTab: tab});
+    }
+
+    setStyles() {
+      var {currentTab, spotlightStyle, impactStyle, spotlightContainer, impactContainer} = this.state;
```

Comment:
Nitpick: this should probably be const, since you're not going to be manipulating any of these values directly.